doc/src/odin/index_markdown.md

```diff
@@ -0,0 +1,451 @@
+> all flipflops)
+>
+>______________________________________
+
+#### Examples .xml configuration file for `-c`
+
+``` {.sourceCode .xml}
+<config>
+    <verilog_files>
+        <!-- Way of specifying multiple files in a project! -->
+        <verilog_file>verilog_file.v</verilog_file>
+    </verilog_files>
+    <output>
+        <!-- These are the output flags for the project -->
+        <output_type>blif</output_type>
+        <output_path_and_name>./output_file.blif</output_path_and_name>
+        <target>
+            <!-- This is the target device the output is being built for -->
+            <arch_file>fpga_architecture_file.xml</arch_file>
+        </target>
+    </output>
+    <optimizations>
+        <!-- This is where the optimization flags go -->
+    </optimizations>
+    <debug_outputs>
+        <!-- Various debug options -->
+        <debug_output_path>.</debug_output_path>
+        <output_ast_graphs>1</output_ast_graphs>
+        <output_netlist_graphs>1</output_netlist_graphs>
+    </debug_outputs>
+</config>
+```
+>__________________________________
+> **NOTE**
+>
+> Hard blocks can be simulated; given a hardblock named `block` in the
+> architecture file with an instance of it named `instance` in the
+> verilog file, write a C method with signature defined in
+> `SRC/sim_block.h` and compile it with an output filename of
+> `block+instance.so` in the directory you plan to invoke Odin\_II from.
+>
+> When compiling the file, you'll need to specify the following
+> arguments to the compiler (assuming that you're in the SANBOX
+> directory):
+>
+> `cc -I../../libarchfpga_6/include/ -L../../libarchfpga_6 -lvpr_6 -lm --shared -o block+instance.so block.c.`
+>
+> If the netlist generated by Odin II contains the definition of a
+> hardblock which doesn't have a shared object file defined for it in
+> the working directory, Odin II will not work if you specify it to use
+> the simulator with the `-g` or `-t` options.
+>
+>____________________________________________
+
+
+>__________________________________
+> **WARNING**
+>
+> Use of static memory within the simulation code necessitates compiling
+> a distinct shared object file for each instance of the block you wish
+> to simulate. The method signature the simulator expects contains only
+> int and int[] parameters, leaving the code provided to simulate the
+> hard blokc agnostic of the internal Odin II data structures. However,
+> a cycle parameter is included to provide researchers with the ability
+> to delay results of operations performed by the simulation code.
+>
+>________________________________________________
+
+#### Examples vector file for `-t` or `-T`
+
+``` {.sourceCode .none}
+# Example vector input file
+GLOBAL_SIM_BASE_CLK intput_1 input_2 input_3 clk_input
+# Comment
+0 0XA 1011 0XD 0
+0 0XB 0011 0XF 1
+0 0XC 1100 0X2 0
+```
+``` {.sourceCode .none}
+# Example vector output file
+output_1 output_2 
+# Comment
+1011 0Xf
```

Comment:
@jeanlego I tried to make the example more specific and in the note I added that the global clock was necessary in the input. I forget why it's necessary

Reply:
http://www.cs.columbia.edu/~cs6861/sis/blif/index.html
Section Flip flops and latches
> If a latch does not have a controlling clock specified, it is assumed that it is actually controlled by a single global clock.
having this clock there allow the user to have predictable output rather than not

Reply:
you should get markdown linter fyi. line 200 and 201 should have newline in between for example oherwise different mardown compiler will react differently

Reply:
Oh okay, I'll add that because probably not everyone is familiar.

Reply:
can you make sure to cite the blif spec there for future reference